scripts/utils.js

```diff
@@ -12,3 +12,8 @@ export function getPathSegments() {
 export function normalizeString(str) {
   return str.toLowerCase().replace(/ /g, '_');
 }
+
+export function normalizeImage(str) {
+  const imagePath = 'https://main--octoral--aemsites.hlx.page/products/assets/';
```

Comment:
Why not just do something like this in products.js?

const productImage = `/product/assets/${normalizeString(entry.image)}`

Reply:
Here normalizeString has got a different return formula - > replace(/ /g, '_')

And we need - > replace(/_/g, '-')

Plus at the same time we would like to keep the location assignment of the images at one location via leveraging normalizeImage function. Apart from entry.image, we do have entry['type-image'] & entry['title-image'] as well. Hence its better to pass all of these strings into a function rather & getting us the required URL for the images.

Reply:
I think @dave-fink is suggesting using relative image link, without specifying the domain. 

Also, we should sanitize image path per EDS guidelines similar to https://github.com/adobe/helix-importer/blob/main/src/utils/FileUtils.js#L23 & https://github.com/adobe/helix-importer/blob/main/src/utils/FileUtils.js#L14.

Something like
```
export function normalizeImage(str) {
  const imagePath = '/products/assets/';
  return sanitizePath(imagePath + str);
}
```

Reply:
Sure. Leveraging relative links now 

export function normalizeImage(str) {
  const imagePath = '/products/assets/';
  return imagePath + str.toLowerCase().replace(/_/g, '-');
}